genomic_data_service/region_indexer.py

```diff
@@ -19,6 +20,10 @@
     'chr18', 'chr19', 'chr20', 'chr21', 'chr22', 'chrX', 'chrY'
 ]
 
+ENCODE_DOMAIN = 'https://test.encodeproject.org'
+REGULOME_ENCODE_ACCESSIONS_MAPPING_PATH = 'regulome_encode_accessions_mapping.pickle'
```

Comment:
why are we using pickle?

Reply:
the pickle contains a hashmap where keys are regulome accessions and values are their correspondent encode accessions.

it's a bit large: ~4MB

they come from the dump of the `residents_regionset` index of Regulome's ES, and then matched to Encode using aliases when applicable and adjusting the Duke and Jolma paths. This process takes some time, that's why I decided to turn into pickle files (they are in this PR). I can also convert them to a spreadsheet.